### PR TITLE
CLDR-14718 improve message when clicking a hidden xpath

### PR DESCRIPTION
**Pull request by srl295**
CLDR-14718

Improve err message when navigating to an xpath that is not visible. (the old message did not mention the actual XPath in question, and removed it from the URL)

image

### CODE REVIEW

**Comment by srl295** [comment]
Also turn this (otherwise nearly hidden) console message into a popup warning:

image

File: tools/cldr-apps/js/src/esm/cldrText.js [hunk 1]
```diff
@@ -412,7 +412,7 @@ const strings = {
   E_NOT_STARTED:
     "The SurveyTool is still starting up. Please wait a bit and hit Reload.",
   E_SPECIAL_SECTION:
-    "An error occurred while trying to ${what}, the server said that those items aren't visible in the Survey Tool.\nPerhaps the URL is incorrect or an item was deprected?",
+    "An error occurred while trying to ${what}.\nThe XPath “${surveyCurrentId}” in locale “${surveyCurrentLocale}” is not visible in the SurveyTool.\nPerhaps the URL is incorrect or an item was deprecated?",
```
Comment (macchiati) [inline tools/cldr-apps/js/src/esm/cldrText.js:415]: A question: "is not visible" in the Survey Tool sounds like "it exists but the ST can't display it". Is that what you meant?

File: tools/cldr-apps/js/src/esm/cldrLoad.js [hunk 2]
```diff
@@ -279,7 +280,9 @@ function verifyJson(json, subkey) {
     return false;
   } else if (json.err_code) {
     var msg_fmt = cldrRetry.format(json, subkey);
-    var loadingChunk = cldrDom.createChunk(msg_fmt, "p", "errCodeMsg");
+    var loadingChunk = cldrDom.createChunk("", "p", "errCodeMsg");
+    loadingChunk.appendChild(cldrDom.createChunk("","i","glyphicon glyphicon-remove-sign fgred"));
+    loadingChunk.appendChild(cldrDom.createChunk(msg_fmt));
```
Comment (btangmu) [inline tools/cldr-apps/js/src/esm/cldrLoad.js:285]: needs formatting

**Review by btangmu — COMMENTED** [review]
js needs formatting

**Comment by srl295** [comment]
> A question: "is not visible" in the Survey Tool sounds like "it exists but
> the ST can't display it". Is that what you meant?
>
Exactly

**Comment by macchiati** [comment]
ok.

Mark

**Review by macchiati — DISMISSED** [review]

File: tools/cldr-apps/js/src/esm/cldrLoad.js (hunk 2 above)
Comment (srl295) [reply tools/cldr-apps/js/src/esm/cldrLoad.js:285]: done

File: tools/cldr-apps/js/src/esm/cldrText.js (hunk 1 above)
Comment (srl295) [reply tools/cldr-apps/js/src/esm/cldrText.js:415]: yes

**Review by macchiati — APPROVED** [review]

**Pull request merged**